### DISCUSSION TITLE
Ignore svg tags everywhere

### DISCUSSION THREAD

**Discussion by thewh1teagle** [prettier/prettier#16295] [Q&A] [answered]
Hey!

Does it possible to ignore all svg tags so it will always be compact like the first one? I would like to configure it on the entire project.
`VSCode`, `TypeScript`, `React`

![image](https://github.com/prettier/prettier/assets/61390950/0d1cb3b9-520c-4d0f-b55e-0c6e08d58a6b)

**Comment by kachkaev** [comment, marked as answer]
👋 @thewh1teagle! I’m afraid that there is no such option. See [Prettier’s Option philosophy](https://prettier.io/docs/en/option-philosophy) and [Why Prettier?](https://prettier.io/docs/en/option-philosophy) for context.
